.travis.yml

```
@@ -10,15 +10,27 @@ matrix:
       os: linux
       env: CONTAINER=centos7
       services: docker
-    - name: "Fedora 28"
+    - name: "Centos 8"
+      os: linux
+      env: CONTAINER=centos8
+      services: docker
+     - name: "Fedora 28"
       os: linux
       env: CONTAINER=fedora28
       services: docker
     - name: "Fedora 29"
       os: linux
       env: CONTAINER=fedora28
       services: docker
```

Both of these have reached their EOL. Thus, I think we are safe to remove them.
If not, we should at least fix the F29 one to actually built that rather than F28.
I.e.
```suggestion
     - name: "Fedora 28"
      os: linux
      env: CONTAINER=fedora28
      services: docker
    - name: "Fedora 29"
      os: linux
      env: CONTAINER=fedora29
      services: docker
```